docs/_components/icon-list.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Icon List
+lead: An icon list reinforces the meaning and visibility of individual list items with a leading icon.
+---
+
+{% include helpers/base-component.html component="icon-list" %}
+
+## Default
+
+{% capture example %}
+    <ul class="usa-icon-list usa-icon-list--size-md">
+  <li class="usa-icon-list__item">
+    <div class="usa-icon-list__icon text-green">
+      <svg class="usa-icon" aria-hidden="true" role="img">
+        <use xlink:href="/assets/img/sprite.svg#check_circle"></use>
```

Comment:
This image is a 404 on the preview site:

https://federalist-340d8801-aa16-4df5-ad22-b1e3e731098e.sites.pages.cloud.gov/preview/18f/identity-design-system/charley/lg-10522-add-icon-list-component/icon-list/

Can we add `{{ site.baseurl }}` so that it uses the correct path? It should be automatically [removed](https://github.com/18F/identity-design-system/blob/f3560e49ac6225677cf554314eea6cdf111cfe20/docs/_includes/helpers/code-example.html#L9) from the copyable snippet.

```suggestion
        <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#check_circle"></use>
```

Reply:
The URL for the `main` version is pretty stable, we have it in the specs:

https://github.com/18F/identity-design-system/blob/f3560e49ac6225677cf554314eea6cdf111cfe20/test/screenshot.test.js#L12-L13

It changes for individual branches though, which is how we can use it for sharing previews to specific pull requests.

It's not super clear, but you can find it by clicking the "Details" link in the checks at the bottom of a pull request.

![Screenshot 2023-08-09 at 4 58 48 PM](https://github.com/18F/identity-design-system/assets/1779930/ef050f7a-217c-45d3-b759-c80fd6921d97)

I'm not sure I'd add it to the documentation, though. Or at least, we recently went through an effort to [discontinue a public-facing site](https://docs.google.com/document/d/1vmrgaUoFtEbIGnzdGkY0lQH58rKfseS9ro76o31ggQI/edit), so I might worry how much we broadcast this as an "official" website for the design system. But it's definitely useful for previewing specific changes from a pull request.

Reply:
Ah, excellent thanks.
👍 URL is stable
👍 Context on the preview site/publicity.